package/models.py

```diff
@@ -375,6 +380,24 @@ def fetch_pypi_data(self):
                 if self.pypi_downloads is None:
                     self.pypi_downloads = total_downloads
 
+                # get documents_url from pypi
+                if self.documentation_url is None:
+                    if info["project_urls"].get("Documentation"):
```

Comment:
After looking at the PyPI API, it seems like the name of this field (`"Documentation"`) can be anything. So, I think we can try all the common variants as well. e.g: `docs`